gitlab-rc.json

```diff
@@ -0,0 +1,51 @@
+{
+	"GitLab CE": {
+		"containers": {
+			"gitlab-ce": {
+				"image": "gitlab/gitlab-ce",
+				"launch_order": 1,
+				"ports": {
+					"80": {
+						"host_default": 8080,
+						"protocol": "tcp",
+						"label": "HTTP WebUI port",
+						"description": "GitLab WebUI port.Suggested default: 8080",
+						"ui":true
+					},
+					"443": {
+						"host_default": 8443,
+						"protocol": "tcp",
+						"label": "HTTPS WebUI port",
+						"description": "GitLab WebUI port.Suggested default: 8443. Not enabled in docker image by default"
+					    },
+					"22": {
+						"host_default": 2222,
+						"protocol": "tcp",
+						"label": "SSH Gitlab port",
+						"description": "GitLab SSH port. Suggested default: 2222"
+					}
+				},
+			"volumes": {
+				"/var/opt/gitlab": {
+					"description": "Choose a Share for GitLab repositories. Eg: create a Share called gitlab-repos for this purpose alone.",
+					"label": "Repository Storage",
+					"min_size": 1073741824
```

Comment:
Is this line correct (line 32) correct? ? According to spec min_size is in KB. I think 1074 GB (1073741824 KB) is too large.